src/main/java/bot/utils/NameProcessor.java

```diff
@@ -0,0 +1,65 @@
+package bot.utils;
+
+import java.util.*;
+
+public class NameProcessor {
+    private NameProcessor() {}
+
+    private static Map<String, String> s;
```

Comment:
You can use `ImmutableMap.of()` here instead of a static block.

Reply:
Seems like a good idea but you can only put up to 5 key-value pairs in the `ImmutableMap.of()` 
https://guava.dev/releases/21.0/api/docs/com/google/common/collect/ImmutableMap.html#of-K-V-K-V-K-V-K-V-K-V-

Reply:
I'm trying to build it using `ImmutableMap.builder()`